impeller/playground/playground_test.cc

```diff
@@ -25,6 +26,8 @@ void PlaygroundTest::SetUp() {
     return;
   }
 
+  ImpellerValidationErrorsSetFatal(true);
```

Comment:
Will this also be set during golden tests?

Reply:
Yes. The golden tests use the same playgrounds harnesses.